05_test_v1a3.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -xe
+
+# shellcheck disable=SC1091
+source lib/logging.sh
+# shellcheck disable=SC1091
+source lib/common.sh
+
+# Disable SSH trong authentication
+export ANSIBLE_HOST_KEY_CHECKING=False
+
+ANSIBLE_FORCE_COLOR=true ansible-playbook \
+    -e "metal3_dir=$SCRIPTDIR" \
+    -e "IMAGE_OS=$IMAGE_OS" \
+    -e "DEFAULT_HOSTS_MEMORY=$DEFAULT_HOSTS_MEMORY" \
+    -e "CAPI_VERSION=$CAPI_VERSION" \
+    -i vm-setup/inventory.ini \
+    -b -vvv vm-setup/v1a3_integration_test.yml
```

Comment:
why are you duplicating the whole ansible play ? you can just parameterize the existing v1alpha2 and change the CR templates, having two sets, one for v1alpha2 and one for v1alpha3. Then depending on the variable you use one or the other

Reply:
The version independence of scripts and playbooks should be implemented in a separate PR. There's a risk that v1a2 CI runs break.